### PR TITLE
[Profitonomy-164] Profile page wallet update

### PR DESCRIPTION
**Pull request by dyudyunov**
Wallet name validation change;
Handle 422 response;
Localization for the profile page.

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-164

**Author concerns:** This PR also should fix [SKILLONOMY-152.](https://youtrack.raccoongang.com/issue/SKILLONOMY-152) because it consists of updates for localizations for cms.

### CODE REVIEW

File: common/djangoapps/edeos/utils.py [hunk 1]
```diff
@@ -150,10 +149,8 @@ def validate_wallets_data(wallet_name, profitonomy_public_key):
               ```
     """
     return (
-        # At least one digit (up to 5) and one letter
-        bool(re.match('^(?=.*[0-5]$)(?=.*[a-z])', wallet_name.strip()))
```
Comment (OPersian) [inline common/djangoapps/edeos/utils.py]: Why was the `At least one digit (up to 5) and one letter` check removed?
As per https://youtrack.raccoongang.com/issue/SKILLONOMY-133, 

```
Cases - expected FAILURE
ddddddddddddd // letters only, no numbers
111111111111 // numbers only, no letters
```  
(this is also mentioned in https://youtrack.raccoongang.com/issue/SKILLONOMY-164);

```
"Имя аккаунта EOS должно быть ровно 12 символов и состоять из строчных букв и цифр до 5"
```

File: common/djangoapps/edeos/views.py [hunk 2]
```diff
@@ -49,11 +49,13 @@ def update_wallet(request):
                 "base_url": getattr(settings, 'EDEOS_API_URL')
             }
             response = send_edeos_api_request(**edeos_post_data)
-            if response:
+            if response and not isinstance(response, int):
                 profile = UserProfile.objects.filter(user=request.user).first()
                 if profile:
                     profile.save_profitonomy_public_key(profitonomy_public_key)
                     profile.save_wallet_name(wallet_name)
+            elif response == 422:
```
Comment (OPersian) [inline common/djangoapps/edeos/views.py]: Could use `httplib.UNPROCESSABLE_ENTITY` instead of `422`

File: lms/static/js/i18n/ro/djangojs.js [hunk 3]
```diff
@@ -5,1882 +5,17 @@
...
   
-  django.pluralidx = function (n) {
-    var v=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n%100>=11 && n%100<=14)? 2 : 3);
-    if (typeof(v) == 'boolean') {
-      return v ? 1 : 0;
-    } else {
-      return v;
-    }
-  };
+  django.pluralidx = function (count) { return (count == 1) ? 0 : 1; };
```
Comment (OPersian) [inline lms/static/js/i18n/ro/djangojs.js]: There's probably no need to re-generate messages automatically, 
as the standard flow seems to be broken/unmanageable for now:
automatic messages generation always gives too many diffs, meaning that i10n/i18n in the codebase does not correspond to the po files. It's been soo for a while, and this needs fixing (within a separate task I guess).

I believe it's OK to just add new strings to po/js files manually and automatically regenerate `mo` files (compilemessages) only for now.

File: common/djangoapps/edeos/utils.py (hunk 1 above)
Comment (OPersian) [reply common/djangoapps/edeos/utils.py]: There also are the tests in the `edeos` app to check:
https://github.com/raccoongang/edx-platform/blob/686b0313b9cf6117089f6506177e09acfd7dc01c/common/djangoapps/edeos/tests/utils.py

File: lms/static/js/i18n/ro/djangojs.js (hunk 3 above)
Comment (OPersian) [reply lms/static/js/i18n/ro/djangojs.js]: это вообще глобальная проблема, давай обсудим

File: common/djangoapps/edeos/utils.py (hunk 1 above)
Comment (dyudyunov) [reply common/djangoapps/edeos/utils.py]: Successs case was chenged: 
```
new_case: studywallet2 or 23451study15 //letters and numbers (numbers are from 1 to 5 inclusive) order of numbers and letters doesn't matter
new_case: 123451234511 //numbers only, no latters
new_case: studywallett //letters only, no numbers
```

File: lms/static/js/i18n/ro/djangojs.js (hunk 3 above)
Comment (OPersian) [reply lms/static/js/i18n/ro/djangojs.js]: Fix for https://youtrack.raccoongang.com/issue/SKILLONOMY-152

File: common/djangoapps/edeos/utils.py (hunk 1 above)
Comment (dyudyunov) [reply common/djangoapps/edeos/utils.py]: tests were checked too.

File: common/djangoapps/edeos/views.py (hunk 2 above)
Comment (dyudyunov) [reply common/djangoapps/edeos/views.py]: Agree, fixed.

File: lms/static/js/i18n/ro/djangojs.js (hunk 3 above)
Comment (dyudyunov) [reply lms/static/js/i18n/ro/djangojs.js]: Only 'ru' translations was added to PR.

**Pull request merged**